lib/rexml/source.rb

```diff
@@ -240,7 +250,9 @@ def ensure_buffer
     # Note: When specifying a string for 'pattern', it must not include '>' except in the following formats:
     # - ">"
     # - "XXX>" (X is any string excluding '>')
-    def match( pattern, cons=false, term: nil )
+    def match( pattern, cons=false )
```

Comment:
Style. Very minor: the spaces around the argument list does not match the other methods in the file.

Reply:
Yes. We should use a consistent coding style in the same code base. But it's out-of-scope of this pull request. We should work on it separately.